shell.nix

```diff
@@ -22,9 +22,6 @@ pkgs.llvmPackages_11.stdenv.mkDerivation {
     # openssh
     pkgs.openssl_1_1
 
-    # Old openssl
-    pkgs.xorg.makedepend
```

Comment:
Again: Not sure if it works without :D Is it maybe preinstalled on some distros and others it is not?